test/unit/gateways/orbital_test.rb

```diff
@@ -174,6 +174,22 @@ def test_network_tokenization_credit_card_data
     end.respond_with(successful_purchase_response)
   end
 
+  def test_schema_for_soft_descriptors_with_network_tokenization_credit_card_data
+    options = @options.merge(
+      soft_descriptors: {
+        merchant_name: 'Merch',
+        product_description: 'Description',
+        merchant_email: 'email@example'
+      }
+    )
+    stub_comms do
+      @gateway.purchase(50, network_tokenization_credit_card(nil, eci: '5', transaction_id: 'BwABB4JRdgAAAAAAiFF2AAAAAAA='), options)
+    end.check_request do |endpoint, data, headers|
```

Comment:
With some of the recent RuboCop clean up that I've done, this should be updated to

```suggestion
    end.check_request do |_endpoint, data, _headers|
```

Reply:
tsk tsk tsk, I think I even reviewed that one! 🤦‍♀️  Thanks for the catch!